bin/dcos-deploy.sh

```diff
@@ -92,7 +92,7 @@ cat <<EOF > lasp-server.json
   "ports": [0, 0],
   "env": {
     "LASP_BRANCH": "$LASP_BRANCH",
-    "AD_COUNTER_SIM_SERVER": "true",
+    "SIMPLE_SIM_SERVER": "true",
```

Comment:
Looks like this will permanently change the behavior of the docs-deploy script.  Maybe we should parameterize this from the environment instead and assume a default of false at the top of the file?